content/en/account_management/rbac/role_api.md

```diff
@@ -218,6 +218,26 @@ The UUIDs for the permissions are as follows:
 {{% /tab %}}
 {{< /tabs >}}
 
+## Granting Permissions within limited scopes
+
+Certain permissions can be granted within a limited scope. This is easily done manually from the Datadog application in [the Pipelines Page][1], but can be done programmatically via the API the Role API if the correct "scope" is added in the payload. The following permissions can be granted within a limited scope:
```

Comment:
```suggestion
Certain permissions can be granted within a limited scope. This can be done manually from the Datadog application in [the Pipelines Page][1], or programmatically via the Role API if the correct "scope" is added in the payload. The following permissions can be granted within a limited scope:
```